satpy/etc/composites/visir.yaml

```diff
@@ -15,6 +15,9 @@ modifiers:
     modifier: !!python/name:satpy.modifiers.EffectiveSolarPathLengthCorrector
     optional_prerequisites:
       - solar_zenith_angle
+    # max_sza None means no reduction beyond correction_limit, see https://github.com/pytroll/satpy/issues/3096
+    max_sza: !!null
+    correction_limit: 150 # use "raw" Li/Shibata without correction
```

Comment:
This change will impact any composite that uses this default modifier, so not only the FCI Cloud type and Cloud Phase RGBs. Given the discussion during and past the RGB workshop I guess that's fine and even preferred. But in that case I suggest to modify the PR title and description to make this more clear.

alternatively make a local modifier inside `./satpy/etc/composites/fci.yaml` and then we change the default in a separate PR, potentially the one I want to do at some point refactoring the code for the standard and Li and Shibata sunz corrections.